### PR TITLE
Change Chinese comment to English comment

### PR DESCRIPTION
**Pull request by throcean**
We should add English comments so that we can get more developers to participate in our projects. I have added some comments and need some project members to review.

### CODE REVIEW

**Comment by yedf2** [comment]
can you review the check results? c-format check fails

File: examples/chat.cc [hunk 1]
```diff
@@ -27,24 +27,24 @@ int main(int argc, const char *argv[]) {
             }
         });
         con->onMsg(new LineCodec, [&](const TcpConnPtr &con, Slice msg) {
-            if (msg.size() == 0) {  //忽略空消息
+            if (msg.size() == 0) {  //Ignore empty messages
```
Comment (jovan-wong) [inline examples/chat.cc]: `if(msg.empty()) {`

File: raw-examples/kqueue.cc [hunk 2]
```diff
@@ -129,7 +130,7 @@ int main() {
     printf("fd %d listening at %d\n", listenfd, port);
     setNonBlock(listenfd);
     updateEvents(epollfd, listenfd, kReadEvent, false);
-    for (;;) {  //实际应用应当注册信号处理函数，退出时清理资源
+    for (;;) {  //The actual application should register the signal processing function and clean up the resources when exiting.
```
Comment (jovan-wong) [inline raw-examples/kqueue.cc]: `while(true) {` will be more making sense.

File: examples/reconnect.cc [hunk 3]
```diff
@@ -7,7 +7,7 @@ int main(int argc, const char *argv[]) {
     Signal::signal(SIGINT, [&] { base.exit(); });
     TcpServerPtr svr = TcpServer::startServer(&base, "", 2099);
     exitif(svr == NULL, "start tcp server failed");
-    svr->onConnState([&](const TcpConnPtr &con) {  // 200ms后关闭连接
+    svr->onConnState([&](const TcpConnPtr &con) {  // Close the connection after 200ms
```
Comment (jovan-wong) [inline examples/reconnect.cc]: Close the connection after waiting 200ms.

File: handy/codec.h [hunk 4]
```diff
@@ -4,22 +4,22 @@
...
+    // > 0 Parse the complete message, put the message in msg, return the number of bytes scanned
+    // == 0 Parsing part of the message
+    // < 0 Parsing error
     virtual int tryDecode(Slice data, Slice &msg) = 0;
     virtual void encode(Slice msg, Buffer &buf) = 0;
     virtual CodecBase *clone() = 0;
 };
 
-//以\r\n结尾的消息
+//Message ending in \r\n
```
Comment (jovan-wong) [inline handy/codec.h]: Messge ended with ?

File: handy/codec.h [hunk 5]
```diff
@@ -4,22 +4,22 @@
...
 struct LineCodec : public CodecBase {
     int tryDecode(Slice data, Slice &msg) override;
     void encode(Slice msg, Buffer &buf) override;
     CodecBase *clone() override { return new LineCodec(); }
 };
 
-//给出长度的消息
+//Give length message
```
Comment (jovan-wong) [inline handy/codec.h]: Given length message

**Comment by jovan-wong** [comment]
This PR will fix [#49](https://github.com/yedf/handy/issues/49).

**Pull request closed without merging**